src/PythonMigrationViewExtension/PythonMigrationAssistantViewModel.cs

```diff
@@ -8,11 +10,13 @@ internal class PythonMigrationAssistantViewModel
         public string OldCode { get; set; }
         public string NewCode { get; set; }
         private PythonNode PythonNode { get; set; }
+        public DynamoViewModel DynamoViewModel { get; set; }
 
-        public PythonMigrationAssistantViewModel(PythonNode pythonNode)
+        public PythonMigrationAssistantViewModel(PythonNode pythonNode, DynamoViewModel dynamoViewModel)
```

Comment:
This change worries me.

1. it would be best not to change a public method - even if I wouldn't really consider this the API.
2. Worse, we should not be using access to the viewModel if we don't need it.

I would really appreciate instead expanding the extensions API instead to allow saving of graphs... and I'm pretty sure thats already possible using commands.

Reply:
yep, looking at this now

Reply:
@mjkkirschner I've changed the migration assistant viewmodel to depend on `WorkspaceModel` and `IPathManager` not `DynamoViewModel`